db/migrate/20210202052337_migrate_variant_unit_values.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateVariantUnitValues < ActiveRecord::Migration
+  def up
+    Spree::Variant.all.select { |v|
+      v.unit_value.nil? && v.product&.variant_unit == "items"
+    }.each do |variant|
+      variant.unit_value = 1
+      variant.save
```

Comment:
I guess we need some callbacks to be executed, right? If not, running a single update statement using `#update_all` would be best.

Reply:
I think we do want the callbacks; I at least erred on the side of running them here since there's quite a few of them and I don't feel like I fully understand what's happening with all of them.